### PR TITLE
GoogleAnalyticsExperiment - Vulnerability fixes 

### PR DESCRIPTION
**Pull request by v-mohithgc**
### Fixed latest CG vulnerabilities for GoogleAnalyticsExperiment Extension 

- [x] run npm audit fix
- [x] Bump version of affected components and its corresponding integrity value
- [ ] To-do : need to test changes

### CODE REVIEW

**Comment by v-mohithgc** [comment]
@rajnish-byte @v-hinti @rvairavelu can you please review the changes

File: Extensions/GoogleAnalyticsExperiments/Src/UIContribution/package-lock.json [hunk 1]
```diff
@@ -7332,7 +7332,7 @@
 			"integrity": "sha512-mbKkMdQKsjX4BAL4bRYTj21edOf8cN7XHdYUJEe+Zn99hVEYcMvKPct1IqNe7+AZPirn8BCDOQBHQZknqmKlZQ==",
 			"dev": true,
 			"requires": {
-				"kind-of": "^3.2.0"
+				"kind-of": "^6.0.2"
```
Comment (rajnish-byte) [inline Extensions/GoogleAnalyticsExperiments/Src/UIContribution/package-lock.json:7335]: just check if this too can be updated to 6.0.3 to be consistent.

**Review by rajnish-byte — APPROVED** [review]

**Comment by mpodriezov** [comment]
@v-mohithgc All seems to fine, but please build and run tests. You have change major version for some libs and we need to make sure we didn't break anything.

**Review by PhilipsonJoseph — APPROVED** [review]

**Pull request closed without merging**